plasma/root_chain/contracts/RootChain/RootChain.sol

```diff
@@ -97,6 +121,7 @@ contract RootChain {
         public
         payable
     {
+        require(currentDepositBlock < 1000);
```

Comment:
Limits the number of deposits per child chain block to 999?

Reply:
Yes. But we can change that to 10k or 100k or 1M. uint256 is large enough.